.travis.yml

```diff
@@ -94,7 +90,7 @@ jobs:
 
     - name: E2E tests
       php: "7.3"
-      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs,eslint,xmllint,phpsyntax,phpunit PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=
+      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs,eslint,xmllint,phpsyntax,phpunit
```

Comment:
This is like https://github.com/WordPress/gutenberg/pull/20547. It's because of a failed `E2E` Travis job.